src/parse/asp/interpreter.go

```diff
@@ -66,6 +66,12 @@ func (i *interpreter) LoadBuiltins(filename string, contents []byte, statements
 		return err
 	} else if len(contents) != 0 {
 		stmts, err := i.parser.ParseData(contents, filename)
+		for _, stmt := range stmts {
```

Comment:
This got lost in the embedding switchover - moving the code to here so we only do this to the actual builtins (to mimic previous behaviour)